solutions/lottery_solution/scripts/test_attack.move

```diff
@@ -0,0 +1,21 @@
+script {
+    use std::signer::address_of;
+    use ctf::lottery::{play, has_won};
+
+    const E_lose :u64 =1;
+
+    fun attack(caller:&signer){
+        
+
+        play(caller);
+
+        let has_won_after =has_won(address_of(caller));
+
+        if(!has_won_after){
```

Comment:
how about we replace this with an assert instead? 

smth like: assert!(has_won(caller), err_code);